BUILD

```diff
@@ -2317,6 +2317,81 @@ grpc_cc_library(
     tags = ["no_windows"],
 )
 
+grpc_cc_library(
```

Comment:
No, you need to use the new grpc_upb_proto_library for this.

Reply:
Done